client/src/components/common/InputField/index.jsx

```diff
@@ -8,12 +8,13 @@ const propTypes = {
   id: PropTypes.string,
   labelText: PropTypes.string,
   type: PropTypes.string.isRequired,
-  name: PropTypes.string.isRequired,
+  name: PropTypes.string,
   value: PropTypes.string,
   placeholder: PropTypes.string,
   className: PropTypes.string.isRequired,
   onChange: PropTypes.func,
   onClick: PropTypes.func,
+  labelClass: PropTypes.string,
```

Comment:
Why you need `labelClass` in the code? @FarahZaqout

Reply:
Because I need to modify the style of the label, give it indentation, different font size, and from time to time color based on where the input field is called.

Reply:
:+1: